Branch-SDK/src/main/java/io/branch/referral/SystemObserver.java

```diff
@@ -600,18 +600,18 @@ public CoroutineContext getContext() {
 
                 @Override
                 public void resumeWith(@NonNull Object o) {
-                    if (o != null) {
```

Comment:
While this is annotated with `NonNull`, it is still possible for a null to be passed through at runtime. I've intentionally ignored the warning to remove the redundant null check.